NineChronicles.Headless/GraphTypes/StandaloneMutation.cs

```diff
@@ -269,6 +270,55 @@ IConfiguration configuration
                     return tx.Id;
                 }
             );
+
+            Field<NonNullGraphType<TxIdType>>(
+                name: "stageTransaction",
+                description: "Add a new transaction to staging and return TxId",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<StringGraphType>>
+                    {
+                        Name = "payload",
+                        Description = "The hexadecimal of string for staging transaction."
```

Comment:
Because `hexadecimal` is an adjective, `The hexadecimal string` seems more smooth.
```suggestion
                        Description = "The hexadecimal string of the transaction to stage."
```